TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py

```diff
@@ -0,0 +1,543 @@
+"""
+    Wrapper functions for FuncIntent
+    This functions include Onosclidriver and Mininetclidriver driver functions
+"""
+def __init__( self ):
+    self.default = ''
+
+def hostIntent( main,
+                name="",
+                host1="",
```

Comment:
since you need some of these arguments for adding the intent, maybe make them required in the function call? I see you are asserting on them below, but it may be more clear if that is conveyed in the method call